core/app/models/concerns/spree/user_address_book.rb

```diff
@@ -56,17 +56,19 @@ def default_address_attributes=(attributes)
 
       def ship_address=(address)
```

Comment:
@allisonlarson while we're in here would you mind adding docs to this method, including the fact that it does nothing and returns nil if the supplied address is nil?